data/GalaxyStellarMassGasMetallicity/conversion/convertStrom2022.py

```diff
@@ -0,0 +1,80 @@
+from velociraptor.observations.objects import ObservationalData
+
+import unyt
+import numpy as np
+import os
+import sys
+
+# Exec the master cosmology file passed as first argument
+with open(sys.argv[1], "r") as handle:
+    exec(handle.read())
+
+output_directory = "../"
+
+if not os.path.exists(output_directory):
+    os.mkdir(output_directory)
+
+# Meta-data
+comment = "The median stellar mass is 1e10 Msun."
+citation = "Strom et al. (2022)"
+bibcode = "2022ApJ...925..116S"
+plot_as = "line"
+redshift = np.array([2.0, 2.7])
+
+M_star = unyt.unyt_array(np.logspace(9.0, 11.3, 100), units="Msun")
+
+models = {
+    "OH": {"alpha": (8.35, 0.02), "beta": (0.14, 0.05), "label": "12+log10(O/H)"},
+    "NH": {"alpha": (7.07, 0.03), "beta": (0.29, 0.07), "label": "12+log10(N/H)"},
+    "FeH": {"alpha": (-0.69, 0.02), "beta": (0.17, 0.22), "label": "[Fe/H]"},
```

Comment:
It should be
```suggestion
    "FeH": {"alpha": (-0.69, 0.02), "beta": (0.17, 0.05), "label": "[Fe/H]"},
```
according to Table 1 from https://arxiv.org/pdf/2111.06410.pdf